client/src/KeyguardClient.ts

```diff
@@ -206,25 +207,34 @@ export class KeyguardClient {
         return publicResult;
     }
 
-    private _onReject(error: any, id: number, state: any) {
-        const command = state.__command;
-        if (!command) {
-            throw new Error('Invalid state after RPC request');
+    private parseState(state?: ObjectType|null) {
+        if (state) {
+            const command = state.__command;
+            if (command) {
+                delete state.__command;
```

Comment:
Side effect ok?

Reply:
Was done the same way before also: https://github.com/nimiq/keyguard-next/pull/244/files#diff-f7e2d477e26eefd5332b8236f5588184L214